042_factorsofanumber.js

```diff
@@ -0,0 +1,10 @@
+let array = [];

```

Comment:
You do not need to declare array globally. Declare it locally inside of the function.